src/pages/AMM/Pools/components/DepositForm/DepositForm.tsx

```diff
@@ -12,6 +12,7 @@ import { displayMonetaryAmountInUSDFormat, newSafeMonetaryAmount } from '@/commo
 import { Dd, DlGroup, Dt, Flex, TokenInput } from '@/component-library';
 import { AuthCTA } from '@/components';
 import { GOVERNANCE_TOKEN, TRANSACTION_FEE_AMOUNT } from '@/config/relay-chains';
+import WarningBanner from '@/legacy-components/WarningBanner';
```

Comment:
This component has been retired for new work (as have all legacy components). it can be used in legacy code, e.g. the bridge app, but shouldn't be used in new features.

The pattern to follow instead is the price impact modal:
https://github.com/interlay/interbtc-ui/tree/master/src/pages/AMM/Swap/components/PriceImpactModal

Reply:
Ah yes, good point—sorry, hadn't looked at the screenshot properly. We should still avoid using legacy components, but I don't think we have a new version of this one. I would check with Rui to see if he's had to do anything similar with any of the new features.

If not, then I would suggest moving the warning banner from 'legacy-components' to 'components' so that we can refactor when Jay's had a chance to review the UX.

Reply:
@danielsimao

Reply:
Alert :) had a feeling there would be a component for this use case!